serverless.js

```diff
@@ -56,15 +61,16 @@ class Website extends Component {
     const s3 = new aws.S3({ region: inputs.region, credentials: this.context.credentials.aws })
 
     this.context.debug(`Configuring bucket ${inputs.bucketName} for website hosting.`)
-    await configureBucketForHosting(s3, inputs.bucketName)
+
+    await configureBucketForHosting(s3, inputs.bucketName, inputs.indexDocument, inputs.errorDocument)
```

Comment:
I had considered modifying `configureBucketForHosting` to pass a white-listed object with configurable values; but opted to just add some positional arguments  with these overrides.